app/templates/releases/beta.hbs

```diff
@@ -1,12 +1,12 @@
 {{title "Beta"}}
 
-{{#project-listing
-    channel="beta"
-    channelDescription="Features spend at least six weeks in beta before they can be included in a stable release."
-    projects=(array model.ember model.emberData)
-    description=description
-}}
-  
+<ProjectListing
+    @channel="beta"
+    @channelDescription="Features spend at least six weeks in beta before they can be included in a stable release."
+    @projects={{array this.model.ember this.model.emberData}}
+    @description={{this.description}}
```

Comment:
**suggestion (soft-blocking):**

I noticed that the `<ProjectListing>` component doesn't use the argument `@description` at all. Let's delete line 7.

```suggestion
    @projects={{array @model.ember @model.emberData}}
```